sound/soc/sof/topology.c

```diff
@@ -399,6 +399,11 @@ int get_token_dai_type(void *elem, void *object, u32 offset)
 	return 0;
 }
 
+int get_token_void(void *elem, void *object, u32 offset)
```

Comment:
I don't understand the commit message at all. 

"
no need to assign the tuple value anywhere, but the
tuple should be copied to struct snd_sof_widget member.
"

what is the difference between assign and copy? I don't get the issue at all.

Reply:
no luck, this is still very confusing. I don't know the details of the topology and I don't need to learn about them, the patch commit message should be high-level enough to describe the functionality.

In this case, I think using the same 'get_type' naming is very misleading, since the type is irrelevant.

Reply:
this doesn't appear to have changed, and all 3 reviewers didn't like the code, so this probably needs more work?